lib/schedule.js

```diff
@@ -500,4 +497,4 @@ module.exports.Invocation = Invocation;
 module.exports.scheduleJob = scheduleJob;
 module.exports.scheduledJobs = scheduledJobs;
 module.exports.cancelJob = cancelJob;
-module.exports.addDateConvenienceMethods = increment.addDateConvenienceMethods;
+module.exports.CronDate = CronDate;
```

Comment:
@santigimeno, I need to export CronDate to do the testing for CronDate since the previous methods for Date is also tested.

However, after I give a thought on it, the testing should be done on cron-parser. Do you want me to remove it?